tbf-challenge/teamwork-be
# feature(endpoint): create article endpoint [start #45]

#### Pre-Submission Checklist







#### Checklist:





#### What does this PR do?


The PR creates the article endpoint
#### How should this be manually tested? 

 
1.Start the server with `npm start`
2. Post sample data bellow to` http://localhost:3000/api/v1/articles `
```
{ 
"id": 6,
"userId" : 1,
"title" : "Rojitech's Frontend Club",
"image" : "Rojitech",
"content" : "dfdgdgsgsdfdfgfgfd",
"published" : "false",
"createdAt" : "{{CurrentDateTime}}"
}
```
3. This should return the response body in the response console of an API client (eg Postman)
 

##### Any background context you want to provide?


This endpoint is used for posting articles to the database
#### What are the relevant issues?
1. Creating an endpoint for posting article data


#### Screenshots (if appropriate)


![Screenshot from 2022-08-15 13-22-01](https://user-images.githubusercontent.com/68056104/184643404-6b594e85-4448-48a5-9b41-342a752c81d9.png)


#### Does it require any deployments updates ?


https://github.com/tbf-challenge/teamwork-be/pull/56


#### Questions:
